emukit/model_wrappers/gpy_model_wrappers.py

```diff
@@ -96,12 +96,14 @@ def Y(self) -> np.ndarray:
         return self.model.Y
 
 
-class GPyMultiOutputWrapper(IModel, IDifferentiable, ICalculateVarianceReduction):
+class GPyMultiOutputWrapper(IModel, IDifferentiable, ICalculateVarianceReduction, IEntropySearchModel):
```

Comment:
What if I want to use a multiple-output GP with an acquisition that is not ES. Would this require to always have the ES interface?

Reply:
I don't fully understand the comment but this change allows you to use entropy search but does not limit you doing anything else.

Reply:
OK, fine, it is GPy specific so no issues.